CppSamples/UtilityNetwork/PerformValveIsolationTrace/PerformValveIsolationTrace.cpp

```diff
@@ -252,10 +256,13 @@ void PerformValveIsolationTrace::performTrace()
       UtilityCategoryComparison* categoryComparison = new UtilityCategoryComparison(selectedCategory, UtilityCategoryComparisonOperator::Exists, this);
       traceParameters->traceConfiguration()->filter()->setBarriers(categoryComparison);
     }
-    m_utilityNetwork->traceAsync(traceParameters).then(this, [this](QList<UtilityTraceResult*>)
-    {
-      onTraceCompleted_();
-    });
+
+    m_taskCanceler->addTask(
+      m_utilityNetwork->traceAsync(traceParameters).then(this, [this](QList<UtilityTraceResult*>)
+      {
+        onTraceCompleted_();
+      })
+    );
```

Comment:
```suggestion
    m_taskCanceler->addTask(m_utilityNetwork->traceAsync(traceParameters).then(this, [this](QList<UtilityTraceResult*>)
    {
      onTraceCompleted_();
    }));
```

Nit pick, feel free to ignore if you want. This just seems to be how the other ones are formatted.